examples/pom_file_generation/README.md

```diff
@@ -0,0 +1,39 @@
+# Generating `<dependency>` nodes for a POM file
+
+This example shows how you can generate the `<dependencies>` section of a POM
+file using `rules_jvm_external`'s generated targets and the `pom_file` rule from
+`bazel-common`.
+
+To generate the XML for the example Java library, build the `//:my_library_pom`
+target:
+
+```
+$ bazel build //:my_library_pom
+INFO: Analysed target //:my_library_pom (13 packages loaded, 444 targets configured).
```

Comment:
I think just the command is helpful, the rest of the bazel output is mostly noise